draft-packages/select/KaizenDraft/Select/Select.tsx

```diff
@@ -63,6 +129,7 @@ const Control: typeof components.Control = props => (
     {...props}
     className={classNames(styles.control, {
       [styles.focusedControl]: props.isFocused,
+      [styles.disabled]: props.isDisabled,
```

Comment:
disabled was previously unimplemented - it came for free from react-select, which I suppose was close enough to the Kaizen design.